src/frame/getFrameHtmlResponse.ts

```diff
@@ -79,14 +82,19 @@ function getFrameHtmlResponse({
     ? `  <meta property="fc:frame:refresh_period" content="${refreshPeriodToUse.toString()}" />\n`
     : '';
 
+  // Set the xmtp metadata if the corresponding content exists.
+  const xmtpHtml = xmtpMinSupportedVersion
+    ? `  <meta property="of:accepts:xmtp" content="${xmtpMinSupportedVersion}" />\n`
+    : '';
```

Comment:
Why not just `vNext`?

Reply:
VNext would be fine too. Unlike Farcaster we actually have versions of the protocol from the start (their spec has versions but in practice everything is VNext).

That means as Frame servers roll out new features like transactions XMTP clients can tell that they need to upgrade to support the Frame.